ui/src/views/Profile/ProfileView.tsx

```diff
@@ -232,105 +152,24 @@ export default function ProfileView(props: Props): React.ReactElement {
-                      width: "70%",
-                      height: "8rem",
-                      marginTop: "2rem",
-                      display: "flex",
-                      justifyContent: "center",
-                      flexDirection: "column",
-                      alignItems: "center",
-                      flexWrap: "no-wrap",
-                    }}
-                  >
-                    <Box className={classes.inputWrapper}>
-                      <Box sx={{ width: "100%" }}>
-                        <CustomTextField
-                          label={"Name"}
-                          isRequired
-                          name="name"
-                          rules={{
-                            required: true,
+                      <Tooltip title="Profile picture">
+                        <Avatar
+                          src={image ? URL.createObjectURL(image) : ""}
+                          style={{
+                            width: "100px",
+                            height: "100px",
                           }}
-                          error={errors.name}
-                          register={register}
                         />
-                      </Box>
+                      </Tooltip>
                     </Box>
-                    <Box className={classes.buttonsWrapper}>
-                      <Button
-                        variant="contained"
-                        onClick={handleSubmit(onSubmit)}
-                      >
-                        {t(translationKeys.buttons.save)}
-                      </Button>
-                      <Button
-                        color="primary"
-                        variant="outlined"
-                        onClick={onCancel}
-                      >
-                        {t(translationKeys.buttons.cancel)}
-                      </Button>
-                    </Box>
-                  </Box>
-                  <Divider
-                    sx={{
-                      marginTop: "2.5rem",
-                      marginBottom: "2.5rem",
-                      width: "70%",
-                    }}
-                  ></Divider>
-                  <Box className={classes.multiInputWrapper}>
-                    <Box sx={{ width: "100%" }}>
-                      <Controller
-                        control={control}
-                        name="friends"
-                        defaultValue={[]}
-                        render={({ field: { onChange, value } }) => {
-                          return (
-                            <CustomMultiSelect
-                              label={t(translationKeys.addFriends)}
-                              options={users}
-                              onChange={onChange}
-                              selectedValues={value}
-                            />
-                          );
-                        }}
+                    <Box sx={{ width: "40%" }}>
+                      <CustomTextField
```

Comment:
The Textfield is clickable on readonly view, the name does not actually change or anything but I am wondering if we might want to just set the Textfield to disabled there or find some other way to make it not interactable.

Reply:
@hskowron Yeah I have set the readonly prop on the textfield but I can find a way in my next to disable the click event